### PR TITLE
AnalyzeReader - Using native units

### PR DESCRIPTION
**Pull request by dgault**
Using millimetres and milliseconds rather than converting

### CODE REVIEW

**Comment by sbesson** [comment]
Thanks for this. The proposed changes certainly match the [Analyze 7.5 specification](https://rportal.mayo.edu/bir/ANALYZE75.pdf). Sadly, this also means we have been misconverting these metadata values and this PR is additionally fixing 2 bugs for this reader. Thus the configuration files for the `Analyze 7.5` datasets in our full repository also needs to be updated - see https://ci.openmicroscopy.org/job/BIOFORMATS-5.1-merge-repository-subset/79/console. Could you open a corresponding data repository configuration PR?

**Comment by dgault** [comment]
No problems Seb, will have a config PR opened this weekend. Using non default units will require a slightly different solution.

File: components/formats-gpl/src/loci/formats/in/AnalyzeReader.java [hunk 1]
```diff
@@ -349,8 +349,8 @@ protected void initFile(String id) throws FormatException, IOException {
     if (getMetadataOptions().getMetadataLevel() != MetadataLevel.MINIMUM) {
       store.setImageDescription(description, 0);
 
-      Length sizeX = FormatTools.getPhysicalSizeX(voxelWidth * 0.001);
-      Length sizeY = FormatTools.getPhysicalSizeY(voxelHeight * 0.001);
+      Length sizeX = FormatTools.getPhysicalSizeX(voxelWidth, UNITS.MM);
+      Length sizeY = FormatTools.getPhysicalSizeY(voxelHeight, UNITS.MM);
       Length sizeZ =
         FormatTools.getPhysicalSizeZ(sliceThickness * 0.001);
```
Comment (sbesson) [inline components/formats-gpl/src/loci/formats/in/AnalyzeReader.java]: Should this also be fixed since the slice thickness is in mm

**Comment by sbesson** [comment]
Discussed with @dgault. A matching PR will be opened to add logic for testing the stored units in the automated tests. Excluding for now.

**Comment by sbesson** [comment]
As we are reaching stability on the test-suite overhaul work to support units (https://github.com/openmicroscopy/bioformats/pull/2050 and corresponding configuration PR), removing the exclude label.

**Comment by dgault** [comment]
Config PR - https://github.com/openmicroscopy/data_repo_config/pull/30

**Comment by sbesson** [comment]
https://ci.openmicroscopy.org/job/BIOFORMATS-5.1-merge-repository-subset/107/ green with this PR included. Imported a couple of analyze datasets into OMERO with this PR included. The physical size is correctly read from the file. Ready to merge once the full repository tests are green.

**Comment by sbesson** [comment]
--rebased-to #2103

**Pull request merged**